tox.ini

```diff
@@ -103,13 +103,16 @@ basepython = python3
 allowlist_externals =
   make
```

Comment:
`make` would no longer be needed here, right?

Reply:
With the recent changes I didn't need to change the tox.ini anymore so this can be reverted.